airbyte-integrations/connectors/source-harvest/source_harvest/spec.yaml

```diff
@@ -0,0 +1,54 @@
+documentationUrl: https://help.getharvest.com/api-v2/
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  type: object
+  required:
+    - account_id
+    - start_date
```

Comment:
Config Builder doesn't allow for custom start_date names, was adapting connector to builder limitations.

Reply:
@pnilan Please add configuration migration from the previous spec format. This will allow users to avoid re-authentication and re-entering all spec values. Here is [an example](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-ads/source_google_ads/config_migrations.py#L30). Additionally, please don't delete previous parameters to maintain backward compatibility.

Reply:
@tolik0 Thanks for this. Added `config_migrations.py` and tests.